orderable/tests/test_models.py

```diff
@@ -207,3 +209,10 @@ def test_changing_parent(self):
         subtask_2.task = task
         subtask_2.save()
         self.assertSequenceEqual(task.subtask_set.all(), [subtask, subtask_2])
+
+    @given(lists(integers(min_value=1), min_size=1, unique=True))
+    def test_save_subtask(self, sort_orders):
+        task = Task.objects.create()
+
+        for order in sort_orders:
+            SubTask.objects.create(task=task, sort_order=order)
```

Comment:
The minimal failing examples all look like this:

```python
sort_orders = [n, n + 1, 1]
```

The `create` fails in `Orderable._save` when it tries to run `.update(sort_order=F('sort_order') + 1)` on the first two `SubTask` instances.

I think the problem is that the first `SubTask` with `sort_order == n` is incremented to `sort_order == n + 1` which conflicts with the second `SubTask` because of the `unique_together` between `sort_order` and `task`.